packages/manager/modules/config/src/index.js

```diff
@@ -1,7 +1,18 @@
 import _Environment from './environment';
+import {
```

Comment:
importing and just exporting value doesn't suit the need ?

Reply:
Nope, in fact if you want to import `A` and `export const A` it's not possible.